data_generation.py

```diff
@@ -0,0 +1,112 @@
+# helper functions for generating dummy data
+
+import random
+import string
+import sql_functions as sf
+
+def new_id(tableID, table):
+	with sf.create_connection('database.db') as conn:
+		maxID = sf.execute_statement(conn, f'SELECT MAX({tableID}) FROM {table}')
+
+		# empty list filter 2x
+		if maxID == [] or maxID == None:
+			return 1
+		else:
+			maxID = maxID[0]
+		if maxID == [] or maxID == None:
+			return 1
+		else:
+			maxID = maxID[0]
+
+		if type(maxID) == type(None):
+			maxID = 0
+		return maxID+1
+
+def find_id(tableID, table):
+	with sf.create_connection('database.db') as conn:
+		firstID = sf.execute_statement
```

Comment:
This execute_statement should have something in it, right?

Reply:
This was on my in-progress commit, and was removed.